static/app/components/charts/releaseSeries.tsx

```diff
@@ -110,7 +110,8 @@ class ReleaseSeries extends React.Component<Props, State> {
       !isEqual(prevProps.environments, this.props.environments) ||
       !isEqual(prevProps.start, this.props.start) ||
       !isEqual(prevProps.end, this.props.end) ||
-      !isEqual(prevProps.period, this.props.period)
+      !isEqual(prevProps.period, this.props.period) ||
+      !isEqual(prevProps.queryExtra, this.props.queryExtra)
```

Comment:
We need to add `query` here:
https://github.com/getsentry/sentry/blob/0a569c3e19c3e0bd59dfcfe1877727c1f6d2a740/static/app/components/charts/releaseSeries.tsx#L141-L147
That's the bit that controls what vertical release lines will be fetched from the backend.